Src/FluentAssertions/Formatting/FormattedObjectGraph.cs

```diff
@@ -101,7 +101,7 @@ private void FlushCurrentLine()
     {
         if (lineBuilder.Length > 0)
         {
-            AppendSafely(lineBuilderWhitespace + lineBuilder);
+            AppendSafely($"{lineBuilderWhitespace}{lineBuilder}");
```

Comment:
Why start using string interpolation?

Reply:
Because we don't have any missed coverage branches anymore

Reply:
It seems that the [general consensus](https://stackoverflow.com/a/72928810/253961) is that string interpolation is more efficient than simple concatenation.

Reply:
Performance doesn't care about consensus 😉

I didn't immediately spot that using string interpolation moves the null check of `lineBuilder` inside `DefaultInterpolatedStringHandler.AppendFormatted`, which increases the code coverage.
After understanding that, I'm fine with this change.

```cs
using BenchmarkDotNet.Attributes;
using BenchmarkDotNet.Running;
using System.Text;

BenchmarkRunner.Run<StringConcat>();

[ReturnValueValidator]
[MemoryDiagnoser]
public class StringConcat
{
    private readonly StringBuilder lineBuilder = new("foo");

    [Params("    ")]
    public string LineBuilderWhitespace { get; set; }

    [Benchmark(Baseline = true)]
    public string Concat() => LineBuilderWhitespace + lineBuilder;

    [Benchmark]
    public string Interpolate() => $"{LineBuilderWhitespace}{lineBuilder}";

    [Benchmark]
    public string Opt() => string.Create(LineBuilderWhitespace.Length + lineBuilder.Length, (LineBuilderWhitespace, lineBuilder), (c, state) =>
    {
        state.LineBuilderWhitespace.CopyTo(c);
        state.lineBuilder.CopyTo(0, c[state.LineBuilderWhitespace.Length..], state.lineBuilder.Length);
    });
}
```

| Method      | Mean     | Error    | StdDev   | Ratio | Gen0   | Allocated | Alloc Ratio |
|------------ |---------:|---------:|---------:|------:|-------:|----------:|------------:|
| Concat      | 15.71 ns | 0.055 ns | 0.046 ns |  1.00 | 0.0115 |      72 B |        1.00 |
| Interpolate | 47.03 ns | 0.149 ns | 0.132 ns |  2.99 | 0.0114 |      72 B |        1.00 |
| Opt         | 14.83 ns | 0.185 ns | 0.164 ns |  0.94 | 0.0063 |      40 B |        0.56 |